### PR TITLE
Improve current design a bit. 

### PR DESCRIPTION
**Pull request by JonathonDunford**
Big thanks to: https://chrome.google.com/webstore/detail/forkdelta-sleek-theme/paangbeajanjnfkppmgidgadjgpgclld/

#88 

Big thanks to @sethcampbell11

### CODE REVIEW

File: css/black.css [hunk 1]
```diff
@@ -1,20 +1,48 @@
 @import 'https://fonts.googleapis.com/css?family=Open+Sans:400,700';
 
-html,body {
+* {
+	-webkit-font-smoothing: antialiased;
```
Comment (freeatnet) [inline css/black.css]: We should retain subpixel antialiasing on Retina screens,

**Comment by freeatnet** [comment]
Looks good for the purposes of a fresh coat of paint! Some small things that I feel we should address before we ship this:

1. Modal "close" control is no longer visible in the new theme
2. "Danger" button no longer appears red (see "Forget Account" modal)
3. The "Total" field in orders doesn't get the same look as other fields, has white text on light-grey background
4. Text overflows "Balance" tab on smaller screens
5. Orders box partially hidden on smaller screens (in particular, on mobile "buy" and "sell" buttons are not visible)

P.S.: Love the new loaders :100:

**Comment by JonathonDunford** [comment]
Fixed.

**Comment by freeatnet** [comment]
:shipit:

**Pull request merged**